app/modules/event-logger.js

```diff
@@ -14,7 +14,7 @@ export default function logEvent(name, description) {
 		body: JSON.stringify({
 			name,
 			description: JSON.stringify(description),
-			client: 'mobile-wiki'
+			client: `mobile-wiki:${config.APP.version}`
```

Comment:
How about separate (optional) field called `client_version` in the object sent to event-logger? It is 10 minutes of work on the service side and would be more consistent with the approach we have in other applications